integration-tests/Cargo.toml

```diff
@@ -38,7 +38,9 @@ cumulus-primitives-core.workspace = true
 emulated-integration-tests-common.workspace = true
 
 # Local
-pop-runtime.workspace = true
+pop-runtime-devnet.workspace = true
+pop-runtime-testnet.workspace = true
```

Comment:
I dont think we need the testnet runtime included for now, probably just adds to compilation time